excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -443,4 +578,14 @@ public static DataSet getDataSet(Workbook workbook, Sheet sheet, Table table, Ex
         final DataSet dataSet = new XlsDataSet(selectItems, workbook, rowIterator);
         return dataSet;
     }
+
+    private static String getNumericCellValueAsStringUsingBuildinFormat(final CellStyle cellStyle, final double cellValue) {
```

Comment:
Can you maybe rename this method to `getNumericCellValueAsString`? Or maybe just `convertToString`?

Reply:
Done